src/library.js

```diff
@@ -1593,6 +1646,13 @@ const createPilingJs = (rootElement, initOptions = {}) => {
     // Consequential updates that cause new actions to be dispatched
     if (stateUpdates.has('grid')) {
       updateGrid();
+    } else if (stateUpdates.has('arrange')) {
```

Comment:
```suggestion
    }
    if (stateUpdates.has('arrange')) {
```

Why `else if`? The two changes aren't mutually exclusive are they?